interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdHttpInterpreterInitializer.scala

```diff
@@ -0,0 +1,100 @@
+package io.buoyant.namerd.iface
+
+import com.twitter.conversions.time._
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.Stack.Transformer
+import com.twitter.finagle._
+import com.twitter.finagle.buoyant.TlsClientPrep
+import com.twitter.finagle.naming.NameInterpreter
+import com.twitter.finagle.param.HighResTimer
+import com.twitter.finagle.service._
+import com.twitter.logging.Logger
+import com.twitter.util.{NonFatal, Throw}
+import io.buoyant.namer.{InterpreterConfig, InterpreterInitializer}
+
+/**
+ * The namerd interpreter offloads the responsibilities of name resolution to
+ * the namerd service via the namerd HTTP streaming API.  Any namers configured
+ * in this linkerd are not used.
+ */
+class NamerdHttpInterpreterInitializer extends InterpreterInitializer {
+  val configClass = classOf[NamerdHttpInterpreterConfig]
+  override def configId: String = "io.l5d.namerd.http"
+}
+
+object NamerdHttpInterpreterInitializer extends NamerdHttpInterpreterInitializer
+
+case class NamerdHttpInterpreterConfig(
+  dst: Option[Path],
+  namespace: Option[String],
+  retry: Option[Retry],
+  tls: Option[ClientTlsConfig]
+) extends InterpreterConfig {
+
+  @JsonIgnore
+  private[this] val log = Logger.get("namerd-http")
```

Comment:
If we're going to special-case the name, should we just move this into `newInterpreter` and use _label_?  Otherwise, getClass.getName